config.go

```diff
@@ -105,6 +105,9 @@ type ClientConfig struct {
 	// HttpRequestDirector modifies the request before it's sent.
 	// Useful for adding authentication headers, for example
 	HttpRequestDirector func(*http.Request) error
+	// WebtorrentTrackerHttpHeader returns a custom header to be used when dialing a websocket connection
+	// to the tracker. Useful for adding authentication headers
+	WebtorrentTrackerHttpHeader func() (http.Header, error)
```

Comment:
Is it possible to do this without the error return?

Reply:
Yeah sure! So a function that just returns an `http.Header` then?

Reply:
Yeah